frontend/src/wells/views/WellDetail.vue

```diff
@@ -498,7 +498,8 @@ Licensed under the Apache License, Version 2.0 (the "License");
                   { key: 'specific_capacity', label: 'Specific Capacity (L/s/m)' },
                   { key: 'analysis_method', label: 'Analysis Method' },
                   { key: 'comments', label: 'Comments' },
-                  { key: 'internal_comments', label: 'Internal Comments' }
+                  { key: 'internal_comments', label: 'Internal Comments' },
+                  ( userRoles.wells.edit ? [{ key: 'private', label: 'Private' }] : [])
```

Comment:
Oh, are these the aquifer fields? I added the internal_comments thing to it bc comments was already there, but I don't think either exists on aquifers.

Reply:
on aquifer params comments exist but you are correct internal comments doesn't. I have amended, thank you!